src/domain/individualApplication/utils.ts

```diff
@@ -20,7 +20,8 @@ export const getCustomerProfile = (profile: CUSTOMER_PROFILE): CustomerProfileCa
     primaryAddress: profile.primaryAddress,
     primaryPhone: profile.primaryPhone?.phone,
     primaryEmail: profile.primaryEmail?.email,
-    ssn: '', // TODO
+    ssn: '-', // TODO
+    showCustomerNameAsLink: true,
```

Comment:
Just realized, if this is always true, can we get rid of it altogether?

Reply:
It's not always true, it's false on the Application card

Reply:
👌